GUI/GtkExtras.hs

```diff
@@ -18,9 +18,10 @@ import Control.Concurrent.MVar
 #if !(mingw32_HOST_OS || mingw32_TARGET_OS)
 import System.Glib.GError
 import Control.Monad
-#endif
 
+#else
 #include "windows_cconv.h"
+#endif
```

Comment:
Can you flip the condition like this? It's easier to read.

```haskell
#if mingw32_HOST_OS || mingw32_TARGET_OS
#include "windows_cconv.h"
#else
import ...
#endif
```

Reply:
Done!